core/tachograph-reader-core.csproj

```diff
@@ -0,0 +1,17 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>netcoreapp2.0</TargetFramework>
+    <ApplicationIcon />
+    <StartupObject />
+  </PropertyGroup>
+
+  <ItemGroup>
+    <Content Remove="DriverCardData.config" />
+    <Content Remove="VehicleUnitData.config" />
+  </ItemGroup>
+  <ItemGroup>
+    <ProjectReference Include="..\standard\tachograph-reader-lib.csproj" />
```

Comment:
I don't think `core` project should reference `standard` project. If you want to see them all in Visual Studio they all should be referenced in a separate solution file.

Reply:
you have to include it to be able to access the standard library functions in the core project

Reply:
ohh yea, but I think it would make it more clearer if it were `src` folder instead of  `standard`. As here you're using .NET Core so then why .NET Standard...